recipes/pyobjc-core/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pyobjc-core" %}
+{% set version = "3.2.1" %}
+{% set md5 = "df752f952825dd68958c28a9b07967b6" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{version}}.tar.gz
+  url: https://pypi.python.org/packages/7e/03/303a5c7f7c3d3af811eba44b32ef957e570be4d5c5b656c0b44ece6191e0/{{ name }}-{{version}}.tar.gz
```

Comment:
Please replace with the URL in [`example`]( https://github.com/conda-forge/staged-recipes/blob/a0222417ed30935a383f5773ff21d23322247cdd/recipes/example/meta.yaml#L18 ).